server/models/procedures.sql

```diff
@@ -75,4 +75,10 @@ SOURCE server/models/procedures/payroll.sql
   analysis procedures
 */
 SOURCE server/models/procedures/analysis.sql
+
+/*
+  migration process procedures
+*/
+SOURCE server/models/procedures/migration-process.sql
```

Comment:
I don't think you need these here since they aren't useful in the actual code, they are only useful for migrations and this file gets built when we create the database the first time on an installation.

I wonder if it would be better to just change the bash file of the migration script to build this when you want to do a migration and then clean it up afterwards.

Reply:
Maybe the file name wasn't good but to my mind we will need to use those procedures in our migration files if we want to drop a column or index